### DISCUSSION TITLE
[Feature] Multi select node components

### DISCUSSION THREAD

**Discussion by AlexanderYW** [bcakmakoglu/vue-flow#264] [Ideas]
- [ ] Option to select multiple node components
- [ ] Drag to select multiple node components
- [ ] Shift and click to select multiple node components
- [ ] Able to move node components in the selected group
- [ ] Able to delete node components in the selected group

## Feature Request

**Describe the solution you'd like**
Add the functionality to select more than one node component at the same time in canvas, able to move them and delete them

**Comment by bcakmakoglu** [comment]
There's:
* SelectKeyCode (which is handling the selection box, by default Shift + Left Click to create a selection box)
* MultiSelectKeyCode (which handles clicking multiple nodes and selecting them all)

So basically all the features you are asking for are already built-in 😅

**Reply by AlexanderYW** [reply]
Damn thanks!, I totally missed that!

Is it possible to edit that functionality, so it does not require shift before dragging? :D

**Reply by bcakmakoglu** [reply]
You mean a figma like selection?

Currently, no. A selection key has to be present to have the selection box appear.
In the future it will be possible, just not right now, sorry

**Reply by AlexanderYW** [reply]
Yeah you could compare it like that :)

No need to be sorry! just wanted to know :D
